### PR TITLE
feat: Add aspect ratio property

### PR DESCRIPTION
**Pull request by kevin-kp**
This PR exposes the Aspect ratio property that is available in the native SDKs of THEOplayer.

### CODE REVIEW

File: android/src/main/java/com/theoplayer/ReactTHEOplayerView.kt [hunk 1]
```diff
@@ -444,4 +446,21 @@ class ReactTHEOplayerView(private val reactContext: ThemedReactContext) :
...
   }
+
+  fun setAspectRatio(aspectRatio: String?) {
+    if (aspectRatio == this.aspectRatio) {
+      return
+    }
+    this.aspectRatio = aspectRatio
+    val value = getAspectRatioFromString(aspectRatio)
+    playerView?.player?.setAspectRatio(value)
+  }
```
Comment (kot331107) [inline android/src/main/java/com/theoplayer/ReactTHEOplayerView.kt:457]: @kevin-kp you might want to get rid of backing field here
```suggestion
  var aspectRatio: String? 
    set(value) {
      if (field == value) {
        return
      }
      field = value
      getAspectRatioFromString(field).let {
        playerView?.player?.setAspectRatio(it)
      }
    }
```

File: android/src/main/java/com/theoplayer/ReactTHEOplayerView.kt [hunk 2]
```diff
@@ -50,6 +51,7 @@ class ReactTHEOplayerView(private val reactContext: ThemedReactContext) :
   private var paused = false
   private var muted = false
   private var fullscreen = false
+  private var aspectRatio: String? = null
```
Comment (kot331107) [inline android/src/main/java/com/theoplayer/ReactTHEOplayerView.kt:54]:
```suggestion
```

**Comment by tvanlaerhoven** [comment]
@kevin-kp Thanks for your PR! We're currently holding it off until the 2.0.0 is released (shortly). We'll check how it fits in then ...

**Comment by tvanlaerhoven** [comment]
@kevin-kp we'll include it in a 1.8.4 release. The 2.0.0 has had the same addition recently.

**Review by wvanhaevre — APPROVED** [review]

**Review by tvanlaerhoven — APPROVED** [review]

**Pull request merged**